pages/projects/skyhookdm.md

```diff
@@ -53,20 +53,23 @@ We compare the query latencies of filtering a 1.2 billion row dataset via Parque
 The above two plots shows how Parquet (top) stays bottlenecked on the client CPU while Rados Parquet (bottom) distributes CPU usage between the storage nodes and allows scale out.
 
 # Ongoing Work
+* Working on deploying Skyhook in the UNL and SSL clusters.
 
-* Integrating SkyhookDM with [Coffea](https://coffeateam.github.io/coffea/) to offload Nanoevents processing to the storage layer for faster analysis.
-![skyhook-coffea](/assets/images/skyhook-coffea.png){:style="display:block; margin-left: auto; margin-right: auto; width: 75%"}
+* Working on making the Coffea-Skyhook integration more user-friendly.
 
-* A middleware to allow writing Parquet files containing Nanoevents from [ServiceX](https://iris-hep.org/projects/servicex.html) to SkyhookDM via CephFS.
+* Working on joining HEP datasets using DuckDB/Arrow/Fugue.
 
-* Preparing to upstream our `RadosParquetFileFormat` API to the Apache Arrow project.
+* A middleware to allow writing Parquet files containing Nanoevents from [ServiceX](https://iris-hep.org/projects/servicex.html) to SkyhookDM via CephFS.
 
 # Important Links
-* [Github repository](https://github.com/uccross/arrow).
-* Getting started [instructions](https://github.com/uccross/arrow/tree/rados-dataset-dev/cpp/src/arrow/adapters/arrow-rados-cls#getting-started) and [notebook](https://github.com/uccross/arrow/blob/rados-dataset-dev/cpp/src/arrow/adapters/arrow-rados-cls/docs/demo.ipynb).
+* [Github repository](https://github.com/apache/arrow/tree/master/cpp/src/skyhook).
+* Getting started [instructions](https://skyhookdm-arrow.readthedocs.io/en/latest/getting_started.html) and [notebook](https://github.com/uccross/arrow/blob/rados-dataset-dev/cpp/src/arrow/adapters/arrow-rados-cls/docs/demo.ipynb).
 * [Code walkthrough](https://www.youtube.com/watch?v=XfJsnadp18c) video.
 
 ## Announcements
+* March, 2022 - Our paper, Skyhook: Toward an Arrow-Native Storage System, to appear in CCGrid 2022.
+* October, 2021 - [Skyhook is now a part of Apache Arrow !](https://medium.com/@jayjeetc/skyhookdm-is-now-a-part-of-apache-arrow-e5d7b9a810ba)
```

Comment:
Please also add the Voltron Data blog post.

Reply:
Done